packages/osd-monaco/src/ppl/constants.ts

```diff
@@ -0,0 +1,367 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export const ID = 'PPL';
+
+/**
+ * PPL token type lookup table organized by Monaco editor token class
+ * Maps Monaco token types to Sets of ANTLR token types for O(1) lookup performance
+ */
+export const PPL_TOKEN_SETS = {
```

Comment:
what's the difference between this file and the xjson/lexer_rules/opensearchppl?